### ISSUE TITLE
YouPHPTube appliance encoder not reachable when running on AWS

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#1374]
I'm not 100% sure why, but the YouPHPTube encoder is inaccessible when running on AWS. Even with the port open in the firewall, port 445 refuses to respond. As a workaround, I moved it to port 8443 and all seems well now.

This should fix it on an existing server:
```
sed -i "s|445|8443|g" /etc/apache2/ports.conf
sed -i "s|445|8443|g" /etc/apache2/sites-available/encoder.conf
sed -i "s|445|8443|g" /var/www/YouPHPTube-Encoder/videos/configuration.php
```
Then restart Apache:
```
systemctl restart apache2.service
```
Also port 8443 will need to be opened (when running on AWS) and port 445 can be removed from the firewall/security group config.

While I was at it, I also ran the updates and noticed that the config isn't set up to support dynamic row lengths (so the dreaded "Specified key was too long; max key length is 767 bytes" issue occurs). Adding the `/etc/mysql/mariadb.conf.d/99-turnkey-mariadb-workaround.cnf` file (e.g. [as we did in Foodsoft](https://github.com/turnkeylinux-apps/foodsoft/commit/c53c9024026d13d45da0b0f760ff2f6f59dce11a) resolved the issue.

**Comment by JedMeister** [comment]
I think that this should be covered by https://github.com/turnkeylinux-apps/youphptube/pull/4

**Comment by lordforbes** [comment]
Great work JedMeister ... I tried for 4 days to get various versions of YouPHPTube working on various hosts - including Docker, and on my own local machine. Always one thing or another that wouldn't work. This is the best I've seen yet. Encoder encodes, the Streamer plays. I just have a problem with the live stream - I think the new (Avideo)version doesn't support rtmp, and I'm trying to stream via OBS through to YouPHPTube. Depending what I read, I need to install something else (nginx-rtmp?) but I can see port 1935 is listening. Basically - I cant see what's wrong... (see screenhot)
<img width="1346" alt="Screenshot 2020-05-23 at 18 33 31" src="https://user-images.githubusercontent.com/37818741/82736768-ef27ca80-9d23-11ea-8803-147c64784156.png">

**Comment by JedMeister** [comment]
This specific issue has been resolved with the AVideo v16.0 appliance release (FYI YouPHPTube has been rebranded as AVideo). Note that we haven't yet updated the AWSMP appliance yet, but have updated the [TurnKey Hub](https://hub.turnkeylinux.org/) 9you'll need to specify "16.x").

The encoder is now provided via your_domain.com/encoder/ (i.e. via the same ports as the main UI - port 80 for HTTP & 443 for HTTPS).

@lordforbes - The rtsp stream issue may still be an issue... Unfortunately I didn't double check when I released this update. If you have any further feedback related to RTSP/live streaming, then best to discuss it over on it's specific issue; #1396
